stock_orderpoint_automatic_creation/README.rst

```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================================
+Stock Orderpoint Automatic Creation
+===================================
+
+This module generates stock orderpoints automatically for a product when is
```

Comment:
This module  automatically generates stock orderpoints for a product when it is
created, based on the information from the user's company.